ihatemoney/migrations/alembic.ini

```diff
@@ -7,6 +7,7 @@
 # set to 'true' to run the environment during
 # the 'revision' command, regardless of autogenerate
 # revision_environment = false
+script_location = .
```

Comment:
I had to put this to be able to use `alembic`. Not sure if this is coming from me not using it the right way or not.

Reply:
what are the commands you used for alembic ?

Reply:
Not sure, but I think it was something like `alembic revision -m "add bill.import_date field"`, ran in the `migrations` dir.

Reply:
Ok. The command `./manage.py db`¹ (sub)command should work out-of-the-box without requiring patch ; could you confirm ?

If so… I think we should document it better :)

¹ or `ihatemoney db` if using an installed version.

Reply:
Indeed, it seems to work. I completely missed the doc about this command though :/

Reply:
I think there simply is no doc, that is why you missed it  :-)